### PR TITLE
Add Pushbullet service =)

### PR DESCRIPTION
**Pull request by vdeville**
After long night !
I have added Pushbullet service ! 💃

### CODE REVIEW

**Comment by tperalta82** [comment]
+1 on this one!  , I think all add it to my production env after testing it

**Comment by MyTheValentinus** [comment]
Hello @tperalta82
I'm using my version in production and work perfectly, i have just forget log listing in interface but is in database.

**Comment by sadortun** [comment]
@tperalta82  can you confirm if this PR is working properly ?

@MyTheValentinus can you rebase it ? Or resolve the conflicts ? 

Thanks!

**Comment by MyTheValentinus** [comment]
Hello @sadortun it's good

**Comment by hiagodotme** [comment]
Show! 👍 
I've added notifications by telegram too, pull request (#527)

File: src/psm/Util/Install/Installer.php [hunk 1]
```diff
@@ -290,6 +290,9 @@ public function upgrade($version_from, $version_to) {
 		if(version_compare($version_from, '3.2.0', '<')) {
 			$this->upgrade320();
 		}
+        if(version_compare($version_from, '3.3.0', '<')) {
+            $this->upgrade320();
```
Comment (TimZ99) [inline src/psm/Util/Install/Installer.php]: Typo

File: static/css/style.css [hunk 2]
```diff
@@ -671,4 +671,7 @@ legend{
 
 .icon-pushover {
   background-position: -72px -168px;
+}
+.icon-pushbullet{
+	background-position: -240px -120px;
```
Comment (TimZ99) [inline static/css/style.css:676]: You're trying to load an icon that isn't there.

**Review by TimZ99 — CHANGES_REQUESTED** [review]
Super quick review. Had no time for testing, will do this on another moment.

**Pull request closed without merging**